variables.tf

```diff
@@ -553,6 +553,12 @@ variable "resource_enabled" {
   default     = true
 }
 
+variable "role_enabled" {
+  description = "Role scanner enabled."
+  type        = bool
+  default     = true
```

Comment:
Since there aren't any default rules for the Role scanner, should we disable it by default?

Reply:
Yes, disabled Roles scanner.